### ISSUE TITLE
Docker not working

### ISSUE THREAD

**Issue by cemsbr** [kytos/kytos#442]
No more details on that.

**Comment by cemsbr** [comment]
Kytos package cannot be installed in docker. According to pypa/setuptools#836, it seems to be a Python bug. I'm trying a workaround for extending egg_info.

```
creating kytos.egg-info
writing kytos.egg-info/PKG-INFO
writing dependency_links to kytos.egg-info/dependency_links.txt
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/src/kytos/kytos/setup.py", line 304, in <module>
    'Environment :: No Input/Output (Daemon)',
  File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/local/src/kytos/kytos/setup.py", line 93, in run
    super().run()
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 273, in run
    writer(self, ep.name, os.path.join(self.egg_info, ep.name))
  File "/usr/local/lib/python3.6/dist-packages/setuptools/command/egg_info.py", line 669, in write_entries
    ep = cmd.distribution.entry_points
AttributeError: 'Distribution' object has no attribute 'entry_points'
```

**Comment by erickvermot** [comment]
apparently this bug was from before the release of python 3.6 (it mentions 3.6-dev, and the issue was closed in Nov 2016, while 3.6.0 was released in Dez 2016).
The 'allowed failures' lines added in the commit that closes the issue (https://github.com/pypa/setuptools/commit/0f9232cdd78d0e8dc1ed155541d6c732a1b026e9 - 31 Oct 2016) are even no longer present in setuptools master branch anymore, it was removed by commit https://github.com/pypa/setuptools/commit/51b10c39c0f55aaa34697b16d4b6b04805a4c8ec (4 Nov 2016).
So, are you sure you are using the latest versions of python. pip and setuptools?

**Comment by beraldoleal** [comment]
According to @cemsbr this is working now. Please, test and if not, reopen.
